tests/kat_sig.c

```diff
@@ -787,6 +2349,11 @@ OQS_STATUS sig_kat(const char *method_name, bool all) {
 		goto err;
 	}
 
+	// temporarily disabled slh dsa testing to reduce time taken on actions
```

Comment:
Does this actually disable SLH-DSA testing or just limit the number of tests to 1?